README.md

```diff
@@ -0,0 +1,61 @@
+# Google Code-in Leaders
+
+***
+It is a website to show the current leaders from all organizations in GCI.
+***
+
+ * [Installation](https://github.com/learneradarsh/gci-leaders#installation)
+ * [Languages](https://github.com/learneradarsh/gci-leaders#languages)
+ * [Usage](https://github.com/learneradarsh/gci-leaders#usage)
+ * [Getting Involved](https://github.com/learneradarsh/gci-leaders#getting-involved)
+ * [Support](https://github.com/learneradarsh/gci-leaders#support)
+ * [Authors](https://github.com/learneradarsh/gci-leaders#authors)
+ * [License](https://github.com/learneradarsh/gci-leaders#license)
+
+***
+
+### Installation
```

Comment:
Instead of breaks, I'd use an h2 (`##`) here as that provides its own more clean break and is the correct level of header.